include/CL/sycl/vec/detail/vec.hpp

```diff
@@ -16,6 +16,11 @@
 
 namespace cl::sycl::detail {
 
+template <typename, int>
+class vec;
+template <typename DataType, int numElements>
+using __swizzled_base_vec__ = detail::vec<DataType, numElements>;
```

Comment:
Idem to `vec` one ?

Reply:
yeah again it's part of the same hackery to get something that builds and works but isn't spec compliant.